### PR TITLE
perf: optimize session model

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
Don't fetch the event, trigger, connection inside the model, it's already exist on the session memo.

## Linear Ticket
https://linear.app/autokitteh/issue/UI-894/optimize-sessions-convertion

## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [x] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/models/session.model.ts [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
 		state: protoSession.state,
-		triggerName: protoSession.memo.trigger_name,
-		connectionName: protoSession.memo.connection_name,
+		triggerName: protoSession.memo?.trigger_name,
+		entrypoint: {
+			col: protoSession.entrypoint?.col,
+			name: protoSession.entrypoint?.name,
+			path: protoSession.entrypoint?.path,
+			row: protoSession.entrypoint?.row,
+		},
```
Comment (J1za) [inline src/models/session.model.ts:20]: why not  `entrypoint: protoSession.entrypoint` ?

File: src/services/sessions.service.ts [hunk 2]
```diff
@@ -85,35 +83,7 @@ export class SessionsService {
...
-
-				return { data: undefined, error };
-			}
-
-			const sessionConverted = convertSessionProtoToViewerModel(
-				session!,
-				event?.sourceType,
-				event?.destinationName
-			);
+			const sessionConverted = convertSessionProtoToViewerModel(session!);
```
Comment (J1za) [inline src/services/sessions.service.ts:86]: what if the session returned 'undefined'?

File: src/models/session.model.ts (hunk 1 above)
Comment (RonenMars) [reply src/models/session.model.ts:20]: `protoSession.entrypoint` type is **CodeLocation**
`session.entrypoint` type is **SessionEntrypoint**

I improved the code to this:
```
		entrypoint: { ...protoSession.entrypoint },
```

File: src/services/sessions.service.ts (hunk 2 above)
Comment (RonenMars) [reply src/services/sessions.service.ts:86]: Refactored a little bit

**Review by J1za — APPROVED** [review]

**Pull request merged**